Resources/Textures/_Moffstation/PersonalItems/Wearables/silverstar.rsi/meta.json

```
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "taken from tg git at https://github.com/tgstation/tgstation/blob/master/icons/obj/clothing/accessories.dmi. Modified by Huaqas (github).",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon"
+    },
+    {
+      "name": "equipped-NECK",
+      "directions": 4
+    }
+  ]
+}
```

```suggestion
}

```
Missing trailing newline

---

Doesn't really matter

---

CONSISTENCY MATTERS, ROOMBA

---

i do not bend to fate, fate bends to me